docker-compose.yaml

```diff
@@ -219,3 +220,179 @@ services:
       OPENSEARCH_PASSWORD: admin
     depends_on:
       - opensearch
+
+  # Otel demo
```

Comment:
Adding the otel demo images

Reply:
let's add this as a separated docker-compose file and merge them in the docker-compose command, so you can choose to run it or not